tools/pythonpkg/tests/fast/pandas/test_pandas_timestamp.py

```diff
@@ -0,0 +1,22 @@
+import duckdb
+import pandas as pd
+
+def run_pandas_with_tz(timezone):
+    con = duckdb.connect()
+    con.execute("SET TimeZone = '"+timezone+"'")
+    df = pd.DataFrame({"timestamp": [pd.Timestamp("2022-01-01 10:15", tz=timezone)]})
+    duck_df = con.from_df(df).df()
+    print (df['timestamp'].dtype)
+    print(duck_df['timestamp'].dtype)
+    print (df)
+    print(duck_df)
+    assert df.equals(duck_df)
+
+def test_pandas_tz_utc():
+    run_pandas_with_tz('UTC')
+
+def test_pandas_tz_utc():
+    run_pandas_with_tz('CET')
+
+def test_pandas_tz_utc():
+    run_pandas_with_tz('Asia/Kathmandu')
```

Comment:
```suggestion
import duckdb
import pandas as pd
from pytest import mark

@mark.parametrize('timezone', ['UTC', 'CET', 'Asia/Kathmandu'])
def run_pandas_with_tz(timezone):
    con = duckdb.connect()
    con.execute("SET TimeZone = '"+timezone+"'")
    df = pd.DataFrame({"timestamp": [pd.Timestamp("2022-01-01 10:15", tz=timezone)]})
    duck_df = con.from_df(df).df()
    print (df['timestamp'].dtype)
    print(duck_df['timestamp'].dtype)
    print (df)
    print(duck_df)
    assert df.equals(duck_df)
```

Reply:
Ah yes I have seen that syntax, been meaning to introduce it into a test.
It essentially runs this test 3 times right, with the 3 different arguments to timezone?